tools/validators/instance_validator/tests/fake_instances/GOOD/good_translation_units_and_states.yaml

```diff
@@ -18,9 +18,9 @@ CHWS_WDT-17:
   translation:
     return_water_temperature_sensor:
       present_value: "points.return_water_temperature_sensor.present_value"
-      states:
-        OPEN: "1"
-        CLOSED: "2"
+      unit_values:
+        degrees_celsius: "degC"
+        degrees_fahrenheit: "degF"
```

Comment:
do these changes change any tests?  (I see a few test files changed but only one test change)

Reply:
not really, the way the tests are written is to just check if an exception is raised or not, need to update those as well